.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,17 +7,36 @@ assignees: ''
 
 ---
 
-**Environment:**
- - AWS ParallelCluster / CfnCluster version [e.g. aws-parallelcluster-2.5.1]
- - Configuration file (i.e. ~/.parallelcluster/config) without any credentials or personal data.
+If you are reporting an issue with AWS Parallelcluster / CfnCluster please make sure to add the following data in order to facilitate the root cause detection:
+
+**Required Info:**
+ - AWS ParallelCluster version [e.g. 2.9.0]:
+ - Full cluster configuration without any credentials or personal data.
```

Comment:
I would ask at least for the cluster name and optionally for the ARN of the CloudFormation template.